clam_trajectory_controller:
    type: dynamixel_hardware_interface/JointTrajectoryActionController
    dependencies:
        - shoulder_pan_controller
        - gripper_roll_controller
        - shoulder_pitch_controller
        - elbow_roll_controller
        - elbow_pitch_controller
        - wrist_roll_controller
        - wrist_pitch_controller
    joint_trajectory_action_node:
        min_velocity: 0.0
        constraints:
            goal_time: 0.3  # time to let the servos finish before the goal is considered 'reached'
# original goal amount was 0.045 but controllers kept aborting
            shoulder_pan_joint:
                goal: 0.2
            gripper_roll_joint:
                goal: 0.2
            shoulder_pitch_joint:
                goal: 0.2
            elbow_roll_joint:
                goal: 0.2
            elbow_pitch_joint:
                goal: 0.2
            wrist_roll_joint:
                goal: 0.2
            wrist_pitch_joint:
                goal: 0.2



